CHECKENFORCER

```diff
@@ -0,0 +1,3 @@
+format: v0.1-alpha
```

Comment:
Does this file have to be in the root of the repo? Also can it be a hidden file (i.e. start with ".")? I'm trying to help eliminate as much garbage in the root as possible and would prefer this to be under the eng directory .

Reply:
I'm trying to make the solution general so that other teams in Microsoft that have the same problem can use this solution. While we would normally put this in the ```eng``` folder, that is our convention, not something that is broadly adopted by other open source projects.

I could poll both locations in the repo (```/CHECKENFORCER``` first, and then ```eng/CHECKENFORCER```). Thats not too bad and covers both the general case and the azure-sdk preferred approach. There is one wrinkle in polling both locations initially because it adds overhead and could drive up rate limiting. I've put a request into GitHub around a possible bug that they have (missing information in the check_run event payload). If they resolve that issue then the hits for these files become very cachable and I'm less concerned about checking multiple locations.

For the time being I can just relocate the file to ```eng/CHECKENFORCER```. I had named it this way to be consistent with ```CODEOWNERS``` which was some advice I got from a friend at GitHub when I talked about what I was working on.

Reply:
OK - so it now operates on ```eng/CHECKENFORCER``` and the ```enabled: true``` flag is optional (we assume enabled unless explicitly disabled. Merging this now so that our core repositories are all up and running generating load.

Reply:
By the way I'd be fine with this file going into the `.github` folder as well I'm just trying to keep it clutter out of the root.